URA-695/vcf_making.sh

```
@@ -0,0 +1,31 @@
+    bcftools filter -i "FORMAT/DP > 99 && FORMAT/AF > 0.03" - -o sentieon-tnbam_comparison/"${sample}_filtered_v5.0.1.vcf"
+    done
+
+for i in sentieon-tnbam_comparison/*.vcf; do bgzip $i; done
+for i in sentieon-tnbam_comparison/*.vcf.gz; do tabix -p vcf $i; done
+
+rm -r bcftools_isec ; mkdir bcftools_isec
+
+# Create stuff for venn diagrams
+for vcf_file in sentieon-tnbam_v3.2.0/*.vcf.gz
+    do sample=$(basename "$vcf_file" .vcf.gz)
+    bcftools isec -p bcftools_isec/${sample} sentieon-tnbam_comparison/${sample}_filtered_v5.0.1.vcf.gz sentieon-tnbam_comparison/${sample}_filtered_v3.2.0.vcf.gz
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)